src/app/prihlasit/page.tsx

```diff
@@ -53,6 +54,18 @@ const LoginPage: NextPage = () => {
 
         login(response.token);
         router.push(Routes.DASHBOARD);
+      } else if (response) {
+        throw new Error(response?.error);
+      }
+    } catch (error: any) {
+      if (error?.response) {
+        error?.response.json().then((data: { error: string }) => {
+          const serverMessage =
+            data?.error === ERROR_MESSAGE.BAD_CREDENTIALS_EN
+              ? ERROR_MESSAGE.BAD_CREDENTIALS_CS
+              : ERROR_MESSAGE.GENERIC_ERROR;
+          setErrorMessage(serverMessage);
```

Comment:
Handle the generic errors here. If we have response, that should be handled above. This is catch, aka. everything went south :D 

`setErrorMessage(ERROR_MESSAGE.GENERIC_ERROR);`